src/libraries/System.Net.Http/ref/System.Net.Http.csproj

```diff
@@ -6,10 +6,14 @@
     <Compile Include="System.Net.Http.cs" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\System.Net.Quic\ref\System.Net.Quic.csproj" />
     <ProjectReference Include="..\..\System.Runtime\ref\System.Runtime.csproj" />
     <ProjectReference Include="..\..\System.Net.Primitives\ref\System.Net.Primitives.csproj" />
     <ProjectReference Include="..\..\System.Net.Sockets\ref\System.Net.Sockets.csproj" />
     <ProjectReference Include="..\..\System.Net.Security\ref\System.Net.Security.csproj" />
     <ProjectReference Include="..\..\System.Diagnostics.DiagnosticSource\ref\System.Diagnostics.DiagnosticSource.csproj" />
+    <ProjectReference Include="..\..\System.Threading.Channels\ref\System.Threading.Channels.csproj" />
```

Comment:
Are the changes in this proj file needed? At least System.Threading.Channels should not be needed anymore